devservices/config.yml

```diff
@@ -13,6 +13,13 @@ x-sentry-service-config:
         branch: master
         repo_link: https://github.com/getsentry/snuba.git
         mode: containerized
+    snuba-profiling:
```

Comment:
Using this will bring up snuba-profiles-consumer, snuba-profiles-chunks-consumer, and functions-consumer